### PR TITLE
sample(REPORT-363140): WPF .NET core barcode extension sample

### PR DESCRIPTION
**Pull request by ArumugasamiMurugesan**
<table>
<tr>
<td>
Description:
</td>
<td>
WPF .NET core barcode extension sample
</td>
</tr>
<tr>
<td>
Analysis:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Reason for not identifying earlier:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Is it a breaking issue?
</td>
<td>
No
</td>
</tr>
<tr>
<td>
Solution description:
</td>
<td>
WPF .NET core barcode extension sample
</td>
</tr>
<tr>
<td>
Areas affected and ensured:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Test cases:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Tested sample location:
</td>
<td>
NA
</td>
</tr>
</table>

### CODE REVIEW

File: WPF/WPF.NETCore-Barcode-Extension/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+## Learn More
+For detailed documentation and tutorials on integrating barcode extensions with Bold Reports in WPF .NET Core applications, refer to the Bold Reports Documentation.
+
+## Support
+If you encounter any challenges or have inquiries regarding report creation or issues in the reports, please do not hesitate to open a [support ticket](https://support.boldreports.com/support) with us. This allows us to investigate the matter and offer assistance to resolve any issues.
+
+
+
+
+## Programmatically Changing the XML DataSource at Runtime in Blazor
```
Comment (ManoranjanRajendran) [inline WPF/WPF.NETCore-Barcode-Extension/README.md]: Is this correct?

File: WPF/WPF.NETCore-Barcode-Extension/ReportViewerWPFCore/ReportViewerWPFCore.csproj.user
Comment (ManoranjanRajendran) [inline WPF/WPF.NETCore-Barcode-Extension/ReportViewerWPFCore/ReportViewerWPFCore.csproj.user:1]: No need to commit this file.

File: WPF/WPF.NETCore-Barcode-Extension/README.md (hunk 1 above)
Comment (ArumugasamiMurugesan) [reply WPF/WPF.NETCore-Barcode-Extension/README.md]: Removed that line

File: WPF/WPF.NETCore-Barcode-Extension/ReportViewerWPFCore/ReportViewerWPFCore.csproj.user
Comment (ArumugasamiMurugesan) [reply WPF/WPF.NETCore-Barcode-Extension/ReportViewerWPFCore/ReportViewerWPFCore.csproj.user:1]: Deleted this file

**Review by ManoranjanRajendran — APPROVED** [review]
Changes fine

**Pull request merged**